.ci/azure/linux.yml

```diff
@@ -177,6 +177,24 @@ jobs:
     displayName: 'IE CAPITests'
     continueOnError: false
 
+  - script: |
+      wget -P tests/samples_smoke_tests/ https://d30ikxcvcet9xo.cloudfront.net/repositories/openvino/ci_dependencies/test/samples_smoke_tests_data.zip     
```

Comment:
is the data approved to be shared? From my point of view it looks fine, but to double check

Reply:
Yes, it was approved.